tests/test_log.py

```diff
@@ -44,6 +44,10 @@ def test_log_constructor():
     pageUrl = test_log.data.page_url
     assert pageUrl == "https://github.com/apache/flagon/tree/master/docker"
 
+    id = test_log.id
```

Comment:
Might want to make this it's own test just for readability if there were to be errors ever, but it is not that big of a deal since the timestamp and log parsing are tightly coupled.